Pseudo_experiments/create_heatmap.ipynb

```diff
@@ -0,0 +1,1141 @@
NjUX79u0RFBQEAKioqEB2djY8PDxQXFyML7/8EpcvX8aIESOq3QY2gw/xT59jx4s4iIiITF9ycjIGDx5s2ON3T79+/RAXF4fNmzdj1KhRWLRoEd555x0EBQVBo9FgwIABiIiIMCwfFxeH999/H8HBwaioqICbmxs+/vhjw3ovX76MCRMm4MqVK7Czs0NoaCgmTJgAlUoF4O4FJjNmzEBOTg5UKhUCAgKwfv16NGjQoNptUEkPXnpCNUIpV/U+GzxPdATFsyosFR3BNPCjRJaiBXdERzAJd8rVoiOYBFt1uegIJuG/3d8XHUHReGsZIiIiIjPGZpCIiIjIjLEZJCIiIjJjbAaJiIiIzBibQSIiIiIzxmaQiIiIyIyxGSQiIiIyY2wGiYiIiMwYn0Bi5tTXtaIjKN8fd3enP6cqrRAdwSTkHW8sOoJJkPhjJ4tkyZu9y9JddABl455BIiIiIjPGZpCIiIjIjLEZJCIiIjJjbAaJiIiIzBibQSIiIiIzxmaQiIiIyIyxGSQiIiIyY2wGiYiIiMwYm0EiIiIiM8ZmkIiIiMiMsRkkIiIiMmNsBomIiIjMGJtBIiIiomoMHz4c3t7e8PHxga+vL8LDw7Ft2zajZaZNmwZ3d3ccPHiw0vs/+ugjhIeHw9vbGyNGjKg0X1xcjBkzZiAwMBC+vr4YNGgQfvjhB8N8fHw8fHx8jL7c3d3x3nvvGZYJCQlB27ZtjZbJzs6udtvYDBIRERHJEB0djSNHjiArKwsvvPACxo8fj5ycHACAVqtFeno6HB0dkZiYWOm9LVq0wLhx4zB48OAq17148WL89NNP2LRpEw4ePIiwsDCMGTMGt27dAgBERUXhyJEjhq/NmzdDpVIhLCzMaD3vvfee0XLu7u7VbhebQSIiIqrVCgoKkJOTU+mroKDgL63PysoKQ4cOhU6nw6lTpwAAW7ZsgUajwfTp07F9+/ZK6x4wYABCQkLg5ORU5Tpzc3MRHByMxo0bw8LCAoMHD0ZxcTHOnTtX5fJJSUnw9PREu3bt/tI2GG3P/3sNVGPmz5+PkpKSv2VdsbGxf8t6iIiIlG7t2rVYunRppfGYmBiMHTv2kddXVlaGdevWQa1Ww8PDAwCQnJyMPn36IDQ0FHPmzEFKSgpGjhwpe52vvPIKPvroI+Tl5aFRo0bYsGED3Nzc8OSTT1b5/Tdt2oR///vflebmz5+POXPmoGnTpnjppZcwZMiQar83m0ETUlJSwiaOiIjoEQ0bNgy9e/euNO7o6PhI64mPj8fy5cuhVqvh5uaGJUuWwM3NDT///DNOnDiBefPmQa1Wo2/fvkhKSnqkZtDDwwMuLi4IDg6GpaUlHBwcEBcXB2tr60rLZmZmory8vNI2zZ8/H15eXtBoNDhw4IChWayuIWQzSERERLWak5PTQw/PPoqoqChER0dXGk9MTESbNm3g6ekJABg4cCBWrlyJrKwsBAQEyFr3uHHjUK9ePezbtw/16tXD7t278frrr2PDhg144oknjJZNSkpCnz59YGdnZzTu7+9v+P/AwECMGDECW7ZsYTNIRERE9E/RarXIyMiAXq9HYGCgYVylUiE5OVl2M/jrr7/i448/Rv369QHcvTK4RYsW2Ldvn1Ez+Ntvv+HQoUOYMWNGteu0sLCAJEnVLsdmkIiIiOgvSk1NhUqlQlpaGmxsbAzj3333HWbNmoVp06bB2dkZ5eXl0Ov1qKiogF6vR2lpKVQqFTQaDQCgQ4cO+Oqrr/DUU0+hbt262L17N06fPo02bdoYfb/ExES0b9/ecK7iPRcvXsT58+fh4+MDKysrHD58GCtXrqxyT+aD2AwSERER/UXJyckYPHgwXF1djcb79euHuLg4bN68GaNGjcKMGTOwefNmw3y7du3g4uKCnTt3AgDmzp2LBQsWoFevXigpKUHTpk0xc+ZM+Pn5Gd5TUlKCLVu2YMqUKZVy3LlzB/Pnz0dubi5UKhWaNWuGN954A8OGDat2G1SSnP2H9LeLjY195ItB/sp7qhPadtrfur5aSaUSncAkqEorREcwCdlvNBQdwSRI/LGTRbLkr3A5zkZPFB1B0XifQSIiIiIzxmaQiIiIyIyxGSQiIiIyY2wGiYiIiMwYm0EiIiIiM8ZmkIiIiMiMsRkkIiIiMmNsBomIiIjMGJ9AIoiNjc0j30D67Nmzf3sO6bdzf/s6ax1JLzqBSdCVlYmOYBLUtxqLjmASLEtEJzANEnfp0N+AzaAgkydPfuT3/N1PHyEiIiLi3xREREREZozNIBEREZEZYzNIREREZMbYDBIRERGZMTaDRERERGaMzSARERGRGWMzSERERGTG2AwSERERmTE2g0RERERmjM0gERERkRljM0hERERUjeHDh8Pb2xs+Pj7w9fVFeHg4tm3bZrTMtGnT4O7ujoMHD1Z6/40bNxATEwMfHx906tQJCxcuhF6vN8xfu3YNb775Jjp16gQ/Pz+88sorOHnypGFep9Ph008/RUhICHx8fDB06FCj+S1btsDHx8foy9PTE1FRUdVuG5tBIiIiIhmio6Nx5MgRZGVl4YUXXsD48eORk5MDANBqtUhPT4ejoyMSExMrvXfixIkAgN27d2Pjxo3YsWMHvvjiC8P8rFmzcOvWLWRmZmLv3r3w9vbG66+/DkmSAAArVqzAli1bsGrVKmRlZaFjx46IiIiAVqsFAISFheHIkSOGr927d8Pa2hphYWHVbhebQSIiIqJHYGVlhaFDh0Kn0+HUqVMA7u6Z02g0mD59OrZv346CggLD8ufPn8e+ffswadIkODg4wNXVFREREUZNY25uLkJDQ+Ho6AiNRoOBAwfi8uXLhvVkZmZi6NChcHV1hUajwdixY3Hz5k3s2LGjyoypqamws7NDjx49qt0eNoNERERUqxUUFCAnJ6fS1/0N26MoKyvDunXroFar4eHhAQBITk5Gnz59EBoaCjs7O6SkpBiWz87OhoODA1q0aGEY8/LywsWLFw179kaNGoX
```

Comment:
This heatmap looks really good! Me being a naive reader, I don't know how to interpret the heatmap, so I might appreciate a sentence or two of explanation about what I'm looking at within the notebook. But if the heatmaps are being made for a manuscript, then that explanation's probably in the text, so your call.

Reply:
Very good suggestion! I will add this